schema_based_on_diagram.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE patients (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(30) NOT NULL,
+  date_of_birth DATE NOT NULL,
+  PRIMARY KEY(id)
+);
+
```

Comment:
Please create indexes on all the foreign keys. It is the basic solution to increase the performance of queries.

Reply:
Thank you for your advice! I have created indexes on the foreign keys.